File: rendering_api/metal/src/core/debugger/metal_debugger.cpp
#include <core/debugger/gryphn_debugger.h>

// these do nothing because I am too lazy to write a debugger for metal at this point in time
GN_EXPORT gnReturnCode gnCreateDebuggerFn(gnDebugger* debugger, gnInstance* instance, const gnDebuggerInfo_t info) {
    return GN_SUCCESS;
}
GN_EXPORT void gnDestroyDebuggerFn(gnDebugger& instance) {

}
// I will at some point but if you have chosen the fate of writing this on a mac than you should really
// hate yourself

File: projects/apis/metal/src/debugger/metal_debugger.m
#include <debugger/gryphn_debugger.h>

// these do nothing because I am too lazy to write a debugger for metal at this point in time
gnReturnCode gnCreateDebuggerFn(gnDebuggerHandle debugger, gnInstanceHandle instance, const gnDebuggerInfo info) {
    return GN_SUCCESS;
}
void gnDestroyDebuggerFn(gnDebuggerHandle instance) {

}
